pkg/workloads/k8s.go

```diff
@@ -150,7 +165,8 @@ func UserPodContainers(api spec.API) ([]kcore.Container, []kcore.Volume) {
 			containerResourceLimitsList["nvidia.com/gpu"] = *kresource.NewQuantity(container.Compute.GPU, kresource.DecimalSI)
 		}
 
-		containerVolumeMounts := append(defaultVolumeMounts(requiresKubexit), containerMounts...)
+		containerVolumeMounts := containerMounts
```

Comment:
This can be removed along with removing neuron RTD (assuming that was the only reason that the volume mounts changed across containers)